preview_generator/preview/builder/office__libreoffice.py

```diff
@@ -128,11 +128,15 @@ def convert_office_document_to_pdf(
 
                 libreoffice_lock = self._get_libreoffice_lock(cache_path)
                 cache_path_hash = hashlib.md5(cache_path.encode("utf-8")).hexdigest()
+                default_filters_by_mimetype = {"text/html": "-infilter=writerglobal8_HTML"}
+                contains_mimetype = default_filters_by_mimetype.keys().__contains__(mimetype)
+                infilter = default_filters_by_mimetype[mimetype] if contains_mimetype else ""
```

Comment:
I do think, you can do:
```suggestion
                infilter = default_filters_by_mimetype.get(mimetype, "")
```
As the previous issue related to type with **mypy** is because  of the error in previous line which create a set instead of a dictionnary.